### ISSUE TITLE
Update `User.referral_eth_earnings` on Address creation

### ISSUE THREAD

**Issue by timolegros** [hicommonwealth/commonwealth#10368]
## Description
Once #10260 is merged, use the community joined event or address creation event to create a projection which updates the `User.referral_eth_earnings`. This can be done by querying `Referrals.referrer_received_eth_amount` for all of that users addresses.

**Comment by dillchen** [comment]
still necessary @Rotorsoft / @timolegros ?

**Comment by Rotorsoft** [comment]
I don't think this is needed since we are already projecting referral_eth_earnings in `handleReferralFeesDistributed`... @timolegros  to confirm

**Comment by timolegros** [comment]
> I don't think this is needed since we are already projecting referral_eth_earnings in `handleReferralFeesDistributed`... [@timolegros](https://github.com/timolegros?rgh-link-date=2025-03-10T17%3A28%3A00.000Z) to confirm

The purpose of this is to backfill `User.referral_eth_earnings`. Suppose:
1. I have address 0x1 which has accrued 1k WEI in fees
	- These fees could be from a secondary Common account or from referrals directly on-chain
2. I sign in to my Common user with address 0x2 which has accrued no fees.
3. I connect my 0x1 address

After step 3, `User.referral_eth_earnings` would still be 0 even though the true value for the user would be 1k WEI

**Comment by Rotorsoft** [comment]
I see, so any address creation path should consider updating the earned fees.

`models.Address.create` found in:
- CreateCommunity.command
- JoinCommunity.command
- setDiscordBotConfig.command

`models.Address.findOrCreate` found in:
- SignIn.command
- magic.ts
